sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/Config/ServiceManagerStore.cs

```diff
@@ -57,6 +58,17 @@ private IInternalServiceHubContextStore CreateHubContextStore(string connectionS
                 // Serves as a reload token provider only
                 .WithConfiguration(new EmptyConfiguration(_configuration))
                 .WithCallingAssembly();
+
+            if (_options.MessagePackHubProtocol != null)
+            {
+                serviceManagerBuilder.AddHubProtocol(_options.MessagePackHubProtocol);
+            }
+            // Allow isolated-process runtimes such as JS, C#-isolated to enable MessagePack hub protocol
+            else if (string.Equals(_configuration[Constants.AzureSignalRMessagePackHubProtocol], Constants.Enabled, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
OrdinalIgnoreCase is enough?

Follow similar configuration pattern as what kestrel do?
```
"Https": {
    "Enabled": false,
}
```

Reply:
updated